src/edu/csus/ecs/pc2/services/web/ContestService.java

```diff
@@ -153,13 +153,6 @@ public Response setStartime(@Context HttpServletRequest servletRequest, @Context
             return Response.status(Status.BAD_REQUEST).entity("Missing 'starttime' key in /contest request").build();
         }
 
-        // verify the JSON didn't contain any OTHER key/value information
-        if (requestMap.size() != 2) {
-            controller.getLog().log(Log.WARNING, "Contest Service PATCH: JSON input contains illegal extra data: '" + jsonInputString + "'");
-            // return HTTP 400 response code per CLICS spec
-            return Response.status(Status.BAD_REQUEST).entity("Extra data in contest patch request").build();
-        }
-
         // if we get here the JSON is valid and contains exactly one element: starttime
```

Comment:
This comment is no longer accurate; with the removal of the test that rejects the request if the JSON contained other fields, it's no longer necessarily true that the JSON `contains exactly one element`.